publisher/config/schema_test.go

```diff
@@ -136,3 +136,47 @@ func TestSchema(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateSchemas_UploadType(t *testing.T) {
+	tests := []struct {
+		name          string
+		appName       string
+		schemas       UploadArtifactSchemas
+		expectedError error
```

Comment:
Would it make sense to group this and `TestValidateSchemas_AppNameShouldBePkgPrefix` together (only one test function)? The implementation is the same and we can be explicit on what is tested using the `name` field.